main_flow/main_flow.py

```diff
@@ -32,13 +31,13 @@
 
 
 def loadCache():
-    global top_instru_dict
-    global top_ingri_dict
+    global top_instruc_dict
```

Comment:
made some naming changes to make it grammatically correct and more streamlined, hope its ok..

Reply:
perfecto- thanks for covering for my Dyslexia!